blog/_posts/2017-05-23-whats-new-scala-native.md

```diff
@@ -0,0 +1,260 @@
+---
+layout: blog
+post-type: blog
+by: Martin Duhem, Guillaume Massé and Denys Shabalin
+title: "What's new with Scala Native?"
+---
+
+It has been a little bit more than two months since Scala Native 0.1 has been
+released. What's new in Scala Native? What should you expect for future
+releases?
+
+## Scala Native 0.2
+
+In April, [Scala Native 0.2 was released][scala-native-0.2-release]. The main
+focus of this release was to increase the coverage of classes from
+the JDK, such that that more programs can be ported to Scala Native without any further
+effort.
```

Comment:
Could you expand more on what this means for the layman? Let's say someone doesn't know anything about Scala Native at all. What does this mean to them? It means you can use more Java classes that you're already used to using in Scala Native.